### PR TITLE
Add aglio template layout files

### PR DESCRIPTION
**Pull request by kgalli**
This PR adds layout templates for aglio, so the generation of the APIv3 documentation better suits our needs. Due to the fact that we use the documentation on our official site openproject.org I think it is a good idea to store these templates in the openproject repository itself.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://feature-add-api-documentation-layouts.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://feature-add-api-documentation-layouts.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: doc/api_documentation_layouts/openproject-layout.jade [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+            style.
+                /* Highlight.js Theme Tomorrow */
+                .hljs-comment,.hljs-title{color:#8e908c}.hljs-variable,.hljs-attribute,.hljs-tag,.hljs-regexp,.ruby .hljs-constant,.xml .hljs-tag .hljs-title,.xml .hljs-pi,.xml .hljs-doctype,.html .hljs-doctype,.css .hljs-id,.css .hljs-class,.css .hljs-pseudo{color:#c82829}.hljs-number,.hljs-preprocessor,.hljs-pragma,.hljs-built_in,.hljs-literal,.hljs-params,.hljs-constant{color:#f5871f}.ruby .hljs-class .hljs-title,.css .hljs-rules .hljs-attribute{color:#eab700}.hljs-string,.hljs-value,.hljs-inheritance,.hljs-header,.ruby .hljs-symbol,.xml .hljs-cdata{color:#718c00}.css .hljs-hexcolor{color:#3e999f}.hljs-function,.python .hljs-decorator,.python .hljs-title,.ruby .hljs-function .hljs-title,.ruby .hljs-title .hljs-keyword,.perl .hljs-sub,.javascript .hljs-title,.coffeescript .hljs-title{color:#4271ae}.hljs-keyword,.javascript .hljs-function{color:#8959a8}.hljs{display:block;background:white;color:#4d4d4c;padding:.5em}.coffeescript .javascript,.javascript .xml,.tex .hljs-formula,.xml .javascript,.xml .vbscript,.xml .css,.xml .hljs-cdata{opacity:.5}
+            //style
```
Comment (myabc) [inline doc/api_documentation_layouts/openproject-layout.jade]: @kgalli why is most of this template commented out?

Comment (NobodysNightmare) [inline doc/api_documentation_layouts/openproject-layout.jade]: good catch @myabc 

We should probably forward that question to the ones that built the layout. Do they go by a known github-handle?

Comment (myabc) [inline doc/api_documentation_layouts/openproject-layout.jade]: @vcostin @aistrati (?)

Comment (myabc) [inline doc/api_documentation_layouts/openproject-layout.jade]: who converted the template from HTML to Jade?

Comment (kgalli) [inline doc/api_documentation_layouts/openproject-layout.jade]: I think Andrei, right @aistrati?

**Pull request merged**